src/webgpu/web_platform/reftests/create-pattern-data-url.html.ts

```diff
@@ -0,0 +1,23 @@
+// creates a 4x4 pattern
```

Comment:
nit: doesn't need to be `.html.ts`; the (completely unexplained, sorry) convention here was that `.html.ts` files would be just the ones included by script tags into the test html files

Reply:
fixed